mava/configs/env/scenario/hide_and_seek_5_vs_5_fobs.yaml

```diff
@@ -0,0 +1,21 @@
+name: Gigastep
+task_name: hide_and_seek_5_vs_5_fobs
+#the teams must be made up of hider/seeker agents
+task_config:
+  team_0:
+    seeker: 5
```

Comment:
It may be obvious to some people, but can we mention that this means **the number of `type` agents** in different files? 
``` suggestion
    seeker: 5 # Number of seeker agents.
```